inst/editor/src/Shiny-Ui-Elements/nodeInfoFactory.tsx

```diff
@@ -22,58 +22,54 @@ import type { unknownUiFunctionInfo } from "./UnknownUiFunction";
  * @returns Function to build info for a ui node that has the arguments provided
  * by the `Args` parameter.
  */
-export function nodeInfoFactory<Args extends UiArgumentsObject>() {
+export function nodeInfoFactory<Args extends namedArgsObject>() {
   return function makeInfo<
     Name extends string,
     TakesChildren extends boolean,
     Comp extends UiNodeComponent<Args, { TakesChildren: TakesChildren }>,
-    Lib extends string = "Internal",
+    RPackage extends string,
+    PyPackage extends string = "none",
     Cat extends string = "Uncategorized"
   >({
```

Comment:
This is where we've split the "library" field to into two. `r_package` which as-of-now is a rename of `library` and a new `py_package` field which will be used to delineate which nodes are supported in python.